src/coreclr/vm/appdomain.hpp

```diff
@@ -1223,8 +1223,8 @@ class BaseDomain
 
 #ifdef HOST_WINDOWS
     // MethodTable to `typeIndex` map. `typeIndex` is embedded in the code during codegen.
-    // During execution corresponding thread static data blocks are stored in `t_threadStaticBlocks`
-    // array at the `typeIndex`.
+    // During execution corresponding thread static data blocks are stored in `t_NonGCThreadStaticBlocks`
+    // and `t_GCThreadStaticBlocks` array at the `typeIndex`.
     TypeIDMap m_threadStaticBlockTypeIDMap;
```

Comment:
Didn't you say you were going to have a separate `m_GCThreadStaticBlockTypeIDMap`?

Reply:
done.